packages/contracts-core/Makefile

```diff
@@ -0,0 +1,33 @@
+# include .env file and export its env vars
```

Comment:
what's the purpose of this file? looks like it's just 1-liners?

Reply:
yy, basically facilitating the one-liners.  I can remove